protocols/request-response/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Unreleased
```

Comment:
```suggestion
## 0.26.1 - unreleased
```

Reply:
Please also bump the version in the manifest.

Reply:
Can do I was curious how I was supposed to handle that bit.